exercises/sub-list/SubListTest.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+
+class SubListTest: XCTestCase {
+
+    func testEmptyLists() {
+
+        XCTAssertEqual(SublistKind.equal, classifier(listOne: [], listTwo: []) )
+    }
+
+    func testEmptyListWithinNonEmptyList() {
+
+        XCTAssertEqual(SublistKind.sublist, classifier(listOne: [], listTwo: [1, 2, 3]))
```

Comment:
I generally add the enum name explicitly for accessing the enum values in the XCTAssert statements shown above for my clarity.However I've removed them in my latest commit.